docker-compose.yml

```
@@ -0,0 +1,9 @@
+services:
+  db:
+    container_name: maria_db
+    image: docker.io/library/mariadb
+    restart: always
+    environment:
+      MYSQL_ROOT_PASSWORD: example
```

Use .env file to prevent changes

```suggestion
      MYSQL_ROOT_PASSWORD: ${MYSQL_ROOT_PASSWORD}
```